ScoutSuite/providers/aws/rules/rulesets/cis-1.2.0.json

```diff
@@ -0,0 +1,164 @@
+{
+    "about": "This ruleset attempts to cover as many recommendations from the CIS Amazon Web Services Foundation v1.0.0.",
```

Comment:
`v1.2.0`

Reply:
Version updated, will be available in the next commit.